mobile/src/lib/background.js

```diff
@@ -32,7 +32,8 @@ const startIosBackgroundService = () => {
     console.log('BackgroundFetch initiated')
 
     loadState().then(persistedState => {
-      const store = getStore(persistedState)
+      initStore(persistedState)
```

Comment:
Because of the naming. But since this commit, I saw a pattern `configureStore`, and I can admit that `const store = configureStore()` is a way better than `const store = initStore()`!